language/move-prover/bytecode/src/read_write_set_analysis.rs

```diff
@@ -134,8 +139,8 @@ impl ReadWriteSetState {
     pub fn apply_summary(
         &mut self, // caller state
         callee_summary_: &Self,
-        actuals: &[AbsAddr],
         type_actuals: &[Type],
+        temp_indices: &[TempIndex],
```

Comment:
Nit: could we call this `actuals` instead of `temp_indices` (here and elsewhere)? I find it slightly more readable to see that all the components of a call (type actuals, actuals, returns) are included.

Reply:
`temp_indices` is indeed not a very useful name, I'll update that!

Reply:
I also reordered the arguments so that it's clear in the diff that we really just changed the type of `actuals`